### DISCUSSION TITLE
How are we using Spock in Eclipse/IDEA these days?

### DISCUSSION THREAD

**Discussion by cfrensley-cmtl** [spockframework/spock#1564] [General]
I updated my Eclipse version this week to a modern version - my previous version was 2019. I had the Jspresso Spock plugin installed. Very handy! No longer supported or even available.

I have several hundred tests written in Spock, so I clearly want to keep using the system.

How do we integrate with IDE's now?

**Comment by Vampire** [comment]
Here you can find the answer of the developer of the Jspresso Spock plugin: https://stackoverflow.com/questions/55715576/trouble-installing-spock-plugin-into-eclipse-2019.

So to copy it here, come to the light side, use IntelliJ IDEA, it has much better Groovy support including built-in Spock support. :-)
